app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> None:
+    split_command = command.split()
+    if split_command[1] == split_command[2]:
+        return
```

Comment:
There's no error handling here. What if the command does not contain 3 parts? This will lead to an `IndexError`. Consider adding some error handling to make your code more robust. For example, you can check the length of `split_command` and return an error message if it's not 3.